src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="fix">
+        <img
+         src="images\logo.png"
+         alt="logo"
+         class="logo_img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
+            <a href="index.html" class="nav__link is-active">apple</a>
+          </li>
+          <li>
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li>
```